src/main/java/org/reaktivity/nukleus/sse/internal/stream/ServerStreamFactory.java

```diff
@@ -739,6 +716,17 @@ private void handleWindow(
             networkReplyBudget += window.credit();
             networkReplyPadding = window.padding();
 
+            if (networkSlot != NO_SLOT && networkReplyBudget >= networkSlotOffset + networkReplyPadding)
+            {
+                MutableDirectBuffer buffer = bufferPool.buffer(networkSlot);
+                DataFW frame = dataRO.wrap(buffer,  0,  networkSlotOffset);
+                networkReply.accept(frame.typeId(), frame.buffer(), frame.offset(), frame.sizeof());
+                doHttpEnd(networkReply, networkReplyId, frame.trace());
```

Comment:
Would prefer a different signal for deferred `END` than implicitly tying to `networkSlot` please.

Reply:
done